### PR TITLE
chore(docs): improve documentation page on the commands

### PR DESCRIPTION
**Pull request by rspurgeon**
APIOPS-50
APIOPS-52

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,159 @@
-# kced temporary documentation
+# Kong go-apiops documentation
 
-Since the `kced` command in intended to be temporary, so is its documentation...
+The [go-apiops](https://github.com/Kong/go-apiops) library provides a set of tools (validation and transformation) for working with API specifications and [Kong Gateway](https://docs.konghq.com/gateway/latest/) declarative configurations. Conceptually, these tools are intended to be organized into a pipeline of individual steps configured for a particular users needs. The overall purpose of the library is to enable users to build a CICD workflow which deliver APIs from specification to deployment. This pipline design allows users to customize the delivery of APIs based on their specific needs.
```
Comment (Tieske) [inline docs/README.md:3]:
```suggestion
The [go-apiops](https://github.com/Kong/go-apiops) library provides a set of tools (validation and transformation) for working with API specifications and [Kong Gateway](https://docs.konghq.com/gateway/latest/) declarative configurations. Conceptually, these tools are intended to be organized into a pipeline of individual steps configured for a particular users needs. The overall purpose of the library is to enable users to build a CI/CD workflow which deliver APIs from specification to deployment. This pipeline design allows users to customize the delivery of APIs based on their specific needs.
```

File: docs/README.md [hunk 2]
```diff
@@ -1,7 +1,159 @@
...
 
-## command `openapi2kong`
+This project is currently in a public beta preview state. As a result, the tools are available in a temporary command line named `kced`. For installation instructions for the `kced` CLI, see the main [README](../README.md). Before these tools are available in a general release, the tools will be integrated into Kong's declarative management CLI, [deck](https://docs.konghq.com/deck/latest/) and the temporary CLI will be deprecated.
```
Comment (Tieske) [inline docs/README.md:5]:
```suggestion
This project is currently in a public beta preview state. As a result, the tools are available in a temporary command line tool named `kced`. For installation instructions for the `kced` CLI, see the main [README](../README.md). Before these tools are available in a general release, the tools will most likely be integrated into Kong's declarative management CLI, [deck](https://docs.konghq.com/deck/latest/) and the temporary CLI will be deprecated.
```

File: docs/README.md [hunk 3]
```diff
@@ -1,7 +1,159 @@
...
+This document contains usage and examples for the current set of tools available, however, Kong will be expanding the library of available tools leading up to a GA release.
+
+## Commands
+
+---
+### `openapi2kong`
+
+The `openapi2kong` transformation is used to convert an OpenAPI Specification (OAS) to a Kong declarative configuration which can be further used with `deck` to configure a Kong Gateway. [OpenAPI Specifications](https://swagger.io/specification/) allow you to define language-agnostic interfaces to your services. The `openapi2kong` tool allows conversion of those specifications directly into Kong Gateway declarative configurations and includes support for Kong extensions (`x-kong`). For details on the format and conversion features, see the included [annotated example file](learnservice_oas.yml).
+
+For full usage instructions, see the the command help:
```
Comment (Tieske) [inline docs/README.md:16]:
```suggestion
For full usage instructions, see the command help:
```

**Review by Tieske — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (rspurgeon) [reply docs/README.md:5]: @Tieske we don't need "most likely" here.  This is the plan.  If we need to revise later we can do so inline.

Comment (Tieske) [reply docs/README.md:5]: updated the suggestion

**Comment by rspurgeon** [comment]
> in the readme, the "## What is this library?" should also be updated, since it only mentions openapi2kong

Good catch, updated the main README

**Review by Tieske — APPROVED** [review]

**Pull request merged**